test/log.test.ts

```diff
@@ -150,4 +173,22 @@ describe('log', () => {
         log.info('Something to be informed about happened', { hotel: 'restaurant' });
         expect(loggerSpy).toBeCalledWith(LEVELS.INFO, 'Something to be informed about happened', { foo: 'bar', hotel: 'restaurant' }, undefined, { prefix: null, suffix: null });
     });
+
+    // Only Node16+ supports cause
+    if (!process.version.startsWith('v14')) {
```

Comment:
kinda weird to see this, isn't it safe to remove this (the more so that this is a test?)

---
edit: now I see we're testing for Node 14 in CI 👀 is there a reason for that?

Reply:
I already tried to nuke v14 tests in CI, but unfortunately we can't until the meteor migration is fully done 😢